public/app/features/dashboard/components/Inspector/PanelInspector.tsx

```diff
@@ -285,12 +292,16 @@ export class PanelInspector extends PureComponent<Props, State> {
     return (
       <>
         {this.renderStatsTable('Stats', stats)}
-        {dataStats.length && this.renderStatsTable('Data source stats', dataStats)}
+        {this.renderStatsTable('Data source stats', dataStats)}
```

Comment:
maybe... that is what was there before, but it always printed "0" in the UI 🤷‍♂

Reply:
I think it was because it was missing the `> 0` part
https://github.com/grafana/grafana/blob/03e3ddcbdb7da5c7b0db5d712a3ed62bc896c2ba/public/app/features/dashboard/components/Inspector/PanelInspector.tsx#L288